### PR TITLE
[RPS-338] Bloomreach experience + URL rewriter

### PR DESCRIPTION
**Pull request by jacsamell**
Upgraded the project to bloomreach experience project in order to add
the URL rewriter plugin. This requires a maven settings.xml with hippo
maven usernames and passwords. Travis requires this as well.

### CODE REVIEW

File: acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/util/TestContentUrls.java [hunk 1]
```diff
@@ -22,9 +22,11 @@ private void setup() {
 
         add("search", "/search");
 
-        add ("publications overview",
```
Comment (ngta1) [inline acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/util/TestContentUrls.java]: Eagle eye! 👍

File: README.md [hunk 2]
```diff
@@ -77,6 +78,20 @@ mvn verify -f acceptance-tests/pom.xml -Pacceptance-test
 See [Automated Acceptance Tests](#automated-acceptance-tests) section for more details on how to further optimise the
 development cycle when working on acceptance tests.
 
+## Auto-export
+
+When making changes to the CMS or console data, you may want your changes to be persisted in the yaml configuration 
+to be boot strapped into the application on startup. The easiest way to do this is to turn auto-export on in the CMS
+(gray button on bottom left of every CMS page). Make sure you turn it on before making any of the changes and turn it
```
Comment (ngta1) [inline README.md]: ..every CMS page) or in the Console (top menu options).

Comment (jacsamell) [reply README.md]: Nice, didn't know that!

**Comment by ngta1** [comment]
Could also maybe add a short commit summary message to compliment the headline.

**Pull request merged**